src/style.css

```diff
@@ -1,3 +1,89 @@
+html {
+  margin: 0;
+  display: block;
```

Comment:
The `display: block;` is not needed on the `html` element as it is the default display value for this element.